yarn-project/archiver/src/archiver/archiver.test.ts

```diff
@@ -38,13 +40,16 @@ describe('Archiver', () => {
     const rollupTxs = [1, 2, 3].map(makeRollupTx);
 
     publicClient.getBlockNumber.mockResolvedValue(2500n);
+    // logs should be created in order of how archiver syncs.
     publicClient.getLogs
       .mockResolvedValueOnce([makeL2BlockProcessedEvent(100n, 1n)])
       .mockResolvedValueOnce([makeUnverifiedDataEvent(102n, 1n)])
       .mockResolvedValueOnce([makeContractDeployedEvent(104n, 1n)])
+      .mockResolvedValueOnce([makeL1ToL2MessageAddedEvent(101n, 1n)])
       .mockResolvedValueOnce([makeL2BlockProcessedEvent(1100n, 2n), makeL2BlockProcessedEvent(1150n, 3n)])
       .mockResolvedValueOnce([makeUnverifiedDataEvent(1100n, 2n)])
       .mockResolvedValueOnce([makeContractDeployedEvent(1102n, 2n)])
+      .mockResolvedValueOnce([makeL1ToL2MessageAddedEvent(101n, 2n)])
```

Comment:
not sure where we publish L1 -> L2 messages but shouldn't the block number be >= `1100n`?